tools/converters/excel/Converters/StringConverters.cs

```diff
@@ -68,6 +68,25 @@ public static class StringConverters
             throw new InvalidOperationException($"Unable to convert value '{value}' to date.");
         }
 
+        /// <summary>
+        /// Convert 'value' to boolean, or null if allowed.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="nullable"></param>
+        /// <returns></returns>
+        public static Boolean? ConvertToBoolean(this string value, bool nullable = false)
```

Comment:
it might be nice to have support for 'true' and 'false' as well, but not required ofc.

Reply:
oh wait, this is just for the tools, so nvm.

Reply:
It does handle `true` and `false` too